### PR TITLE
Add additional admin setting to match security related group policies

### PR DESCRIPTION
**Pull request by yao-msft**
Added corresponding admin settings for InstallerHashOverride and LocalArchivemalwareScanOverride
Also added tests.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/2772)

### CODE REVIEW

File: src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include "winget/Archive.h"
 #include "winget/Filesystem.h"
```
Comment (ryfu-msft) [inline src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp:6]: Nit: Should these also use <>? I can fix when I create the PR for removing zip experimental feature

Comment (yao-msft) [reply src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp:6]: yes

File: src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp [hunk 2]
```diff
@@ -27,7 +28,8 @@ namespace AppInstaller::CLI::Workflow
             }
             else
             {
-                if (context.Args.Contains(Execution::Args::Type::IgnoreLocalArchiveMalwareScan))
+                if (context.Args.Contains(Execution::Args::Type::IgnoreLocalArchiveMalwareScan) &&
+                    Settings::IsAdminSettingEnabled(Settings::AdminSetting::LocalArchiveMalwareScanOverride))
```
Comment (ryfu-msft) [inline src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp:32]: >         Settings::IsAdminSettingEnabled(Settings::AdminSetting::LocalArchiveMalwareScanOverride)) [](http://example.com/codeflow?start=11&length=98)

Should there be a separate elseif block for when someone includes --ignore-local-archive-malware-scan but does not have the admin setting for LocalArchiveMalwareScanOverride enabled? If so, should there be a different error message specific to admin settings?

**Review by ryfu-msft — APPROVED** [review]

File: src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp (hunk 2 above)
Comment (yao-msft) [reply src/AppInstallerCLICore/Workflows/ArchiveFlow.cpp:32]: This message is handled in command invocation.

**Pull request merged**